From: Sven Eckelmann <sven@narfation.org>
Date: Thu, 28 Sep 2017 18:36:49 +0200
Subject: batctl: upgrade package to latest release 2017.3

 * bugs squashed:
   - Fix error messages on traceroute send failures

Signed-off-by: Sven Eckelmann <sven@narfation.org>

diff --git a/batctl/Makefile b/batctl/Makefile
index 433f0dfc7c781412cf2d9ae467684232948af521..2a1912550ab2251a1999cc3c5b2ec380d6d15f2d 100644
--- a/batctl/Makefile
+++ b/batctl/Makefile
@@ -9,10 +9,10 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=batctl
 
-PKG_VERSION:=2017.2
+PKG_VERSION:=2017.3
 PKG_RELEASE:=0
-PKG_MD5SUM:=be82ff074beb2fdb160b54d4e579fbed
-PKG_HASH:=c460d0910d03916e2cb43b60ca427d0101474680cfc096cc2d28a259a94f536c
+PKG_MD5SUM:=a4219f90b077a874d207cb0973d4d3cf
+PKG_HASH:=29c7f075023e02ed9a9299008ec73cfdb4e1b04991d27e4f396131b4059b88a8
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://downloads.open-mesh.org/batman/releases/batman-adv-$(PKG_VERSION)
