From: Sven Eckelmann <sven@narfation.org>
Date: Sun, 14 Oct 2018 20:03:17 +0200
Subject: mac80211: Increase default distance setting

Signed-off-by: Sven Eckelmann <sven@narfation.org>

Forwarded: no

diff --git a/package/kernel/mac80211/files/lib/netifd/wireless/mac80211.sh b/package/kernel/mac80211/files/lib/netifd/wireless/mac80211.sh
index 6ff627b03813bb2de0cd1f465014cc74031106a3..fdd03142dbe655e5085241f5a781b807fbda3f5e 100644
--- a/package/kernel/mac80211/files/lib/netifd/wireless/mac80211.sh
+++ b/package/kernel/mac80211/files/lib/netifd/wireless/mac80211.sh
@@ -1098,7 +1098,7 @@ drv_mac80211_setup() {
 
 	set_default rxantenna 0xffffffff
 	set_default txantenna 0xffffffff
-	set_default distance 0
+	set_default distance 1000
 
 	[ "$txantenna" = "all" ] && txantenna=0xffffffff
 	[ "$rxantenna" = "all" ] && rxantenna=0xffffffff
